docs/book/component-gallery/model-registries/mlflow.md

```diff
@@ -0,0 +1,143 @@
+
+[MLflow Model Registry](https://mlflow.org/docs/latest/model-registry.html)) is
+a powerful tool that you would typically use in the experimenting, QA and
+production phase to manage and track machine learning model versions. It is
+designed to help teams collaborate on model development and deployment, and keep
+track of which models are being used in which environments. With MLflow Model
+Registry, you can store and manage models, deploy them to different environments,
+and track their performance over time. This tool is useful in the following
+scenarios:
+
+* If you are working on a machine learning project and want to keep track of
+different model versions as they are developed and deployed.
+* If you need to deploy machine learning models to different environments and
+want to keep track of which version is being used in each environment.
+* If you want to monitor and compare the performance of different model versions
+over time, and make data-driven decisions about which models to use in production.
+* If you want to simplify the process of deploying models either to a production
+environment or to a staging environment for testing.
+
+## How do you deploy it?
+
+The MLflow Experiment Tracker flavor is provided by the MLflow ZenML
+integration, you need to install it on your local machine to be able to register
+an MLflow Model Registry component. Note that MLFlow model registry requires
+[MLFlow Experiment Tracker](../experiment-trackers/mlflow.md) to be present in
+the stack.
+
+```shell
+zenml integration install mlflow -y
+```
+
+Once the MLflow integration is installed, you can register an MLflow Model
+Registry component in your stack:
+
+```shell
+# Register the MLflow model registry
+zenml model-registry register mlflow_model_registry --flavor=mlflow
+
+# Register and set a stack with the new model registry
+zenml stack register custom_stack -r mlflow_model_registry ... --set
+```
+
+{% hint style="info" %}
+The MLFlow Model Registry will automatically use the same configuration as the
+MLFlow Experiment Tracker. So if you have a remote MLFlow tracking server
+configured in your stack, the MLFlow Model Registry will also use the same
+configuration.
+{% endhint %}
+
+## How do you use it?
+
+
+### Built-in MLflow Model Registry step
+
+After registering the MLflow Model Registry component in your stack, you can
+use it in a pipeline by using the `mlflow_model_registry_step` which is a
+built-in step that is provided by the MLflow ZenML integration. This step
+automatically register the model that was produced by the previous step in the
+pipeline.
+
+```python
+# Pipeline run with MLflow model registry step
+mlflow_training_pipeline(
+    importer=loader_mnist(),
+    normalizer=normalizer(),
+    trainer=tf_trainer(params=TrainerParameters(epochs=5, lr=0.003)),
+    evaluator=tf_evaluator(),
+    model_register=mlflow_register_model_step(
+        params=MLFlowRegistryParameters(
+            name="Tensorflow-mnist-model",
```

Comment:
What's the best practice here for naming models? First letter in caps? hyphenated? underscores?

Reply:
hmm i would replace all to `tensorflow-mnist-model` I think it`s better